electron/testing-diograph-js-integration.js

```diff
@@ -0,0 +1,11 @@
+const { DiographJson } = require('../diograph-js')
```

Comment:
Could we separate tests/specs clearly from the production code?
- separate tests -folder or similar
- unit specs using .spec -postfix